app/services/distribution_service.rb

```diff
@@ -13,7 +13,7 @@ def perform_distribution_service(&block)
     Rails.logger.error "[!] #{self.class.name} failed because of Insufficient Allotment #{distribution_organization.short_name}: #{distribution.errors.full_messages} [#{e.message}]"
     set_error(e)
   rescue StandardError => e
-    Rails.logger.error "[!] #{self.class.name} failed to destroy distribution for #{distribution_organization.short_name}: #{distribution.errors.full_messages} [#{e.inspect}]"
```

Comment:
I removed "to destroy distribution" because this part of the rescue can catch a range of error scenarios, not particularly destroying a distribution.